free-3.md

```diff
@@ -57,12 +74,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This seems like a project that could actually have a lot of cool language design. That said, a lot of initial work would have to be done to teach or input rules of various poems to the system, and then work would have to be done on the checker, both of which seem to be substantial tasks. After that, though, all sorts of additional functionality and design options open up; would you want to maybe have the language be able to suggest rhymes, alliterations, and other thing? Maybe we would want to allow users to define a new poem type (rhyme scheme, meter, length etc) that could then be checked by the checker? It seems like possibilities abound here.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This seems like an idea that can be implemented to be large or small, and as such seems to be appropriately ambitious. It seems largely extensible and with an MVP that is not outrageous.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+As I described in the previous section, this seems like a good idea for a project because it could be extended to include all sorts of features, but could work well with just a subset of functionality. I can't think of too too many downsides to this one -- perhaps I'm not considering potential drawbacks here... is this even a DSL, though? That may be the biggest question here.
```

Comment:
I really agree. I think this can be a very cool idea!
Overall, I think you will want to think about how you really want your users to interact with this DSL. That should be the biggest question you're asking yourself when choosing between these three project ideas.